guava/pom.xml

```diff
@@ -199,6 +199,7 @@
             <link>https://errorprone.info/api/latest/</link>
           </links>
           <overview>../overview.html</overview>
+          <additionalJOption>${maven-javadoc-plugin.additionalJOptions}</additionalJOption>
```

Comment:
Small fix: was included for Android, but not for JRE. Did not appear to be intentional

Reply:
I think I see it in both "top-level" `pom.xml` files...

- https://github.com/google/guava/blob/0b1c477311b2c7a0a89f5e0ac785084e3ab15d34/pom.xml#L214
- https://github.com/google/guava/blob/0b1c477311b2c7a0a89f5e0ac785084e3ab15d34/android/pom.xml#L219

...and in neither `guava/pom.xml` file...

- https://github.com/google/guava/blob/0b1c477311b2c7a0a89f5e0ac785084e3ab15d34/guava/pom.xml
- https://github.com/google/guava/blob/0b1c477311b2c7a0a89f5e0ac785084e3ab15d34/android/guava/pom.xml

That probably makes sense? Or if we need it here, we probably need it in `android/guava/pom.xml`.